t4.cpp

```diff
@@ -1,4 +1,6 @@
 namespace {
 intt;
 int x = 1.1;
```

Comment:
narrowing conversion from constant 'double' to 'int' [bugprone-narrowing-conversions,cppcoreguidelines-narrowing-conversions]